src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health-pie/health-pie-color.enum.ts

```diff
@@ -1,7 +1,13 @@
 export enum HealthPieColor {
-  DEFAULT_RED = '#ff7592',
-  DEFAULT_BLUE = '#1d699d',
-  DEFAULT_ORANGE = '#ffa500',
-  DEFAULT_MAGENTA = '#564d65',
-  DEFAULT_GREEN = '#00bb00'
+  DEFAULT_RED = '#c9190b',
```

Comment:
Is it possible to use a css class instead of the hex representation of color? This way these colors will also change if the colors for the css definitions change.

Reply:
@Devp00l The only way I found is using Using CSS custom properties (variables):
https://github.com/ceph/ceph/pull/36476/files#diff-ed514dc25f47372b9a4e9115dfae890bR7

Reply:
@Devp00l The only way I found is using CC custom properties (variables):
https://github.com/ceph/ceph/pull/36476/files#diff-ed514dc25f47372b9a4e9115dfae890bR7